src/styles/blocks/main.scss

```diff
@@ -0,0 +1,20 @@
+.main {
+  padding: 50px 0 73px;
+  display: flex;
+
+  &__catalog {
+    width: ( $cardWidth * $cardsAmountInRow +
```

Comment:
you need `min-width`

<img width="585" alt="image" src="https://user-images.githubusercontent.com/55802630/173081889-dd02adfb-44f7-46b6-b302-c347e60f0d84.png">


<img width="419" alt="image" src="https://user-images.githubusercontent.com/55802630/173081929-20dcffe6-740b-4593-8e8a-e26e8228ebf9.png">